extmod/modtls_mbedtls.c

```diff
@@ -210,11 +210,10 @@ STATIC mp_obj_t ssl_context_make_new(const mp_obj_type_t *type_in, size_t n_args
 
     // Create SSLContext object.
     #if MICROPY_PY_SSL_FINALISER
-    mp_obj_ssl_context_t *self = m_new_obj_with_finaliser(mp_obj_ssl_context_t);
+    mp_obj_ssl_context_t *self = mp_obj_malloc_with_finaliser(mp_obj_ssl_context_t, type_in);
     #else
-    mp_obj_ssl_context_t *self = m_new_obj(mp_obj_ssl_context_t);
+    mp_obj_ssl_context_t *self = mp_obj_malloc(mp_obj_ssl_context_t, type_in);
     #endif
```

Comment:
I did think this one and the other like it could just have the `#if MICROPY_PY_SSL_FINALISER` block removed and have just the `mp_obj_malloc_with_finaliser` version here as it falls back to `mp_obj_malloc` if disabled.